<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wifiui</class>
 <widget class="QWidget" name="wifiui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>797</width>
    <height>564</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>BankGothic Md BT</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>IP</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_ip">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Adobe 宋体 Std L</family>
             <pointsize>10</pointsize>
            </font>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>PORT</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_port">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Adobe 宋体 Std L</family>
             <pointsize>10</pointsize>
            </font>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>60</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="pushButton_open">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Maximum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_close">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_Rev">
       <property name="text">
        <string>RECEIVE</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="textEdit_Rev">
       <property name="font">
        <font>
         <family>Adobe 宋体 Std L</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_Send">
           <property name="text">
            <string>Send</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Maximum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>50</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit_Send">
         <property name="font">
          <font>
           <family>Adobe 宋体 Std L</family>
           <pointsize>10</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QComboBox" name="comboBox">
         <item>
          <property name="text">
           <string>Hex</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Utf8</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Maximum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Send</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Maximum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>Quit</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_open</sender>
   <signal>released()</signal>
   <receiver>wifiui</receiver>
   <slot>wifiConnect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>746</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>15</x>
     <y>491</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_close</sender>
   <signal>released()</signal>
   <receiver>wifiui</receiver>
   <slot>wifiClose()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>755</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>496</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_ip</sender>
   <signal>editingFinished()</signal>
   <receiver>wifiui</receiver>
   <slot>editedToSure()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>659</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_port</sender>
   <signal>editingFinished()</signal>
   <receiver>wifiui</receiver>
   <slot>editedToSure()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>689</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>released()</signal>
   <receiver>wifiui</receiver>
   <slot>msgSend()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>772</x>
     <y>433</y>
    </hint>
    <hint type="destinationlabel">
     <x>602</x>
     <y>361</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>released()</signal>
   <receiver>textEdit_Rev</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>567</x>
     <y>317</y>
    </hint>
    <hint type="destinationlabel">
     <x>491</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>released()</signal>
   <receiver>wifiui</receiver>
   <slot>wifiQuit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>523</x>
     <y>544</y>
    </hint>
    <hint type="destinationlabel">
     <x>791</x>
     <y>504</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>wifiConnect()</slot>
  <slot>wifiClose()</slot>
  <slot>editedToSure()</slot>
  <slot>msgToShow()</slot>
  <slot>msgSend()</slot>
  <slot>wifiQuit()</slot>
 </slots>
</ui>
